### DISCUSSION TITLE
Is there a way to get last_four_digits of the card entered within CardPayment brick

### DISCUSSION THREAD

**Discussion by joxpulp** [mercadopago/sdk-js#174] [Q&A] [answered]
I wonder if there is a way to get the last 4 digits of the card entered within CardPayment brick, i know that in secureFields's [createCardToken()](https://github.com/mercadopago/sdk-js/blob/main/API/fields.md#mp-instancefieldscreatecardtokennonpcidata) method response the `last_four_digits` information is available, but in bricks the [getFormData()](https://github.com/mercadopago/sdk-js/blob/main/API/bricks/card-payment.md#brick-controllergetformdata) method does not have this info available, since this information is like an standard and is available in other payment processors like [Stripe](https://stripe.com/docs/api/payment_methods/object#payment_method_object-card-last4) and even in secureFields, and in my case I really need this information to show to the users the `last_four_digits` of the card, seems a little odd to me that this info is not available in cardPayments's brick getFormData() method 😕 😢, thank you in advance for the future response

**Comment by francineemilia** [comment]
Hi @joxpulp! As [answered at React SDK Thread](https://github.com/mercadopago/sdk-react/discussions/64#discussioncomment-6449098) we are currently unable to provide the last four, however we have added your suggestion to our improvement backlog! Thanks for you idea and feedback.

**Reply by joxpulp** [reply]
:S okay, thanks, i noticed that a card token api call is already called automatically in cardPayment bricks, I don't understand why not make it the response object that contains `last_four_digits` property available to us, why this response is only available using secureFields/coreMethods?: 

![Captura de pantalla 2023-07-18 a la(s) 05 08 04 p  m](https://github.com/mercadopago/sdk-js/assets/73070103/44732c72-d300-4d71-821f-367192094eb1)

As a feedback to the team, I think the additionalData/formData/cardData, etc... response returned to us should be standardized, cos this cause friction on us (developers too), now I need to rollback with my cardPayment bricks usage and do it with secureFields/coreMethods just to have one property, thanks again and apologies for any inconvenience, I hope you understand my frustration as well. 🙏🏽

**Reply by icaldana** [reply, marked as answer]
Hello @joxpulp, thank you for the suggestions!

I'd like to clarify that Core Methods and Checkout Bricks are different products with different purposes, thus, it was never desired to replicate all the behaviors from one feature to another. 

Despite that, thinking on use cases like yours, we've added the export of `last_four_digits` into our backlog - as mention by @francineemilia - plus, we are even considering providing all the additional properties that Core Methods exports inside an addition data field.

I hope you keep using our product. 
Thank you again.
